### DISCUSSION TITLE
details: Failed to cudaFree pointer 0x7f38d8000000 (error 1 invalid argument)

### DISCUSSION THREAD

**Discussion by Nebuverse** [milvus-io/milvus#4186] [Q&A and General discussion] [answered]
Why is there insufficient video memory?

**Comment by Nebuverse** [comment]
Faiss assertion 'err == cudaSuccess' failed in void faiss::gpu::freeMemorySpace(faiss::gpu::MemorySpace, void*) at gpu/utils/MemorySpace.cpp:72; details: Failed to cudaFree pointer 0x7f38d8000000 (error 1 invalid argument)

**Comment by shiyu22** [comment, marked as answer]
You can follow [this issue](https://github.com/milvus-io/milvus/issues/3646) and comments and reopen about your scenarios if needed.
